functions/trigger-image-transform-node/serverless.yml

```
@@ -0,0 +1,46 @@
+service: scaleway-image-transform
+configValidationMode: off
+
+provider:
+  name: scaleway
+  runtime: node18
```

Did you try with node 20 ?

```suggestion
  runtime: node20
```